pkg/controller/v1beta1/inferenceservice/reconcilers/ingress/ingress_reconciler.go

```diff
@@ -401,3 +413,13 @@ func routeSemanticEquals(desired, existing *v1alpha3.VirtualService) bool {
 		equality.Semantic.DeepEqual(desired.ObjectMeta.Labels, existing.ObjectMeta.Labels) &&
 		equality.Semantic.DeepEqual(desired.ObjectMeta.Annotations, existing.ObjectMeta.Annotations)
 }
+
+func getHostPrefix(isvc *v1beta1.InferenceService, disableIstioVirtualHost bool) string {
+	if disableIstioVirtualHost == true {
+		if isvc.Spec.Transformer != nil {
+			return fmt.Sprintf("%s-%s-default", isvc.Name, string(constants.Transformer))
+		}
+		return fmt.Sprintf("%s-%s-default", isvc.Name, string(constants.Predictor))
```

Comment:
Same here use the [default predictor name utility](https://github.com/kserve/kserve/blob/master/pkg/constants/constants.go#L411) so we can get rid of the default name later on easier.